discord/state.py

```diff
@@ -281,7 +281,10 @@ def parse_guild_member_remove(self, data):
     def parse_guild_member_update(self, data):
         server = self._get_server(data.get('guild_id'))
         user_id = data['user']['id']
-        member = server.get_member(user_id)
+        try:
+            member = server.get_member(user_id)
+        except AttributeError:
```

Comment:
That is literally impossible. If you look at the code for `server.get_member`, it is **impossible** for it to throw an error (unless you did something crazy and overwrote the server._members to not be a dict).  You may have been using a much older version of the library, or your error may have been due to another reason.  My best guess is that whatever you changed happened to fix it for other reasons.